proto/wippersnapper/uart/v1/uart.proto

```diff
@@ -18,36 +18,36 @@ message UARTBusData {
 }
 
 /**
-* UARTDeviceAttachRequest represents a message sent from IO to a device
+* UARTAdd represents a message sent from IO to a device
 * to configure the UART bus (if not already configured) and attach a device.
 */
-message UARTDeviceAttachRequest {
+message UARTAdd {
   UARTBusData bus_info     = 1; /** The UART bus configuration. */
   string device_id         = 2[(nanopb).max_size = 15]; /** The unique identifier of the device to attach to the UART bus, from Adafruit_WipperSnapper_Components. */
   int32 polling_interval   = 3; /** The polling interval, in milliseconds, to use for the device. */
 }
 
 /**
-* UARTDeviceAttachResponse represents a message sent from a device to IO to
+* UARTAdded represents a message sent from a device to IO to
 * confirm that a device has been attached to the UART bus.
 */
-message UARTDeviceAttachResponse {
+message UARTAdded {
   string device_id = 1[(nanopb).max_size = 15]; /** The unique identifier of the device to attach to the UART bus, from Adafruit_WipperSnapper_Components. */
-  bool is_success  = 2; /** True if the UARTInitRequest was successful, False otherwise. */
+  bool is_success  = 2; /** True if the UARTInit was successful, False otherwise. */
 }
 
 /*
-* UARTDeviceDetachRequest represents a message sent from IO to a device
+* UARTRemove represents a message sent from IO to a device
 * to detach a device from the UART bus.
 */
-message UARTDeviceDetachRequest {
+message UARTRemove {
     string device_id = 1[(nanopb).max_size = 15]; /** The unique identifier of the device to detach from the UART bus. */
 }
 
 /**
-* UARTDeviceEvent represents incoming data from a UART sensor.
+* UARTEvent represents incoming data from a UART sensor.
 */
-message UARTDeviceEvent {
+message UARTEvent {
   string device_id = 1[(nanopb).max_size = 15];                    /** The unique identifier of the device to attach to the UART bus, from Adafruit_WipperSnapper_Components. */
-  repeated wippersnapper.i2c.v1.SensorEvent sensor_event  = 2[(nanopb).max_count = 15]; /** A, optionally repeated, SensorEvent from a sensor. */
+  repeated wippersnapper.sensor.v1.SensorEvent sensor_event  = 2[(nanopb).max_count = 15]; /** A, optionally repeated, SensorEvent from a sensor. */
```

Comment:
Hm, in `sensor.proto`, we don't currently have a `SENSOR_TYPE_BYTES` `SensorType` and `SensorEvent`'s `value` field only supports `float_value`.

This may be useful for storing IR transmitter codes (long-term enhancement) too..

Reply:
Added `SENSOR_TYPE_BYTES` to `SensorType` enum and `bytes_value` to `SensorEvent`